### ISSUE TITLE
Crash when using pattern: '' for footer

### ISSUE THREAD

**Issue by Inrixia** [kamiyo/multi-progress-bars#19]
When using a pattern of '' (empty string) a crash occours.
Example:
```ts
this.mpb?.setFooter({
  message: `\n\n${processed}\n${downloaded}\n${speed}`,
  pattern: '',
})
```

```
C:\Users\Inrix\git\Inrixia\Floatplane-Downloader\node_modules\multi-progress-bars\dist\multi-progress-bars.cjs:421
        const base = clampString(pattern.repeat(Math.ceil(this.logger.width / stringWidth__default['default'](pattern))), this.logger.width);
                                         ^

RangeError: Invalid count value
    at String.repeat (<anonymous>)
    at MultiProgressBars.makeBorder (C:\Users\Inrix\git\Inrixia\Floatplane-Downloader\node_modules\multi-progress-bars\dist\multi-progress-bars.cjs:421:42)
    at MultiProgressBars.setFooter (C:\Users\Inrix\git\Inrixia\Floatplane-Downloader\node_modules\multi-progress-bars\dist\multi-progress-bars.cjs:472:50)
    at Downloader.updateSummaryBar (C:\Users\Inrix\git\Inrixia\Floatplane-Downloader\dist\Downloader.js:69:19)
    at Request.<anonymous> (C:\Users\Inrix\git\Inrixia\Floatplane-Downloader\dist\Downloader.js:137:34)
    at Request.emit (node:events:390:28)
    at IncomingMessage.<anonymous> (C:\Users\Inrix\git\Inrixia\Floatplane-Downloader\node_modules\floatplane\node_modules\got\dist\source\core\index.js:800:18)
    at Object.onceWrapper (node:events:509:28)
    at IncomingMessage.emit (node:events:402:35)
    at IncomingMessage.origin.emit (C:\Users\Inrix\git\Inrixia\Floatplane-Downloader\node_modules\floatplane\node_modules\@szmarczak\http-timer\dist\source\index.js:43:20)
```

**Comment by Inrixia** [comment]
Using " " works ofc, but there should probably be some code to handle a empty string.
Also I found trying to call setFooter with a string instead of a object does not appear to work, but that may need to be a seperate issue.

**Comment by kamiyo** [comment]
Should be fixed in version `4.2.1`. Let me know.

**Comment by Inrixia** [comment]
Yep fixed thanks!
